platforms/r3-corda/configuration/roles/create/certificates/nms/vars/main.yaml

```diff
@@ -1,4 +1,7 @@
 userpassword_networkmap: admin
 mongodbPassword_networkmap: newdbnm
 rootca: "{{playbook_dir}}/build/corda/cordarootca"
-nmsca: "{{playbook_dir}}/build/corda/cordanetworkmap"
\ No newline at end of file
+nmsca: "{{playbook_dir}}/build/corda/cordanetworkmap"
+mongorootca: "{{playbook_dir}}/build/corda/mongorootca"
+mongodbca: "{{playbook_dir}}/build/corda/mongodbca"
+mongodb_cert_subj: "{{ mongoroot_cert_subj }}-{{component_name}}"
```

Comment:
This key can be changed in platforms/r3-corda/configuration/roles/setup/nms/tasks/main.yaml  itself.
Why create a new var again?